src/setting_definitions/__init__.py

```diff
@@ -85,6 +86,15 @@ def vault_required(settings):
                       "Otherwise, just use the default",
                       default_value="https://support.montagu.dide.ic.ac.uk:8200",
                       is_required=vault_required),
+
+    SettingDefinition("notify_channel",
+                      "What slack channel should we post in?",
+                      "e.g., montagu. Leave as the empty string to not post",
+                      default_value=""),
+    SettingDefinition("notify_instance_name",
```

Comment:
Maybe we should just have an `instance_name` setting. Might turn out to be useful elsewhere?

Reply:
For some reason I thought that you wanted to avoid that (or at least a "production" flag).  But I can see how it might come in useful elsewhere (e.g., printed in the portals perhaps)